qemu/tests/qsd_backup_pull.py

```diff
@@ -0,0 +1,300 @@
+import six
+import socket
+import logging
+
+from provider import backup_utils
+from provider import blockdev_base
+from provider import job_utils
+
+from provider.nbd_image_export import InternalNBDExportImage
+from provider.virt_storage.storage_admin import sp_admin
+from provider.qsd import QsdDaemonDev, add_vubp_into_boot
+
+from virttest import qemu_storage
+from virttest import utils_disk
+
+LOG_JOB = logging.getLogger('avocado.test')
```

Comment:
I think we don't need it, we can use self.test.log. Thanks.

Reply:
LOG_JOB is the same logger as self.test.log.
It's OK to use self.test.log, LOG_JOB was introduced *before* because we used a regular expression to replace the 'old logger'

Reply:
Have replaced all LOG_JOB by self.test.log